sirepo/package_data/static/js/sirepo-plotting.js

```diff
@@ -1189,12 +1189,12 @@ SIREPO.app.directive('colorPicker', function(appState, panelState) {
         },
         template: `
             <div>
-                <button class="dropdown-toggle sr-color-button" data-ng-style="bgColorStyle()" data-toggle="dropdown"></button>
+                <button type="button" class="dropdown-toggle sr-color-button" data-ng-style="bgColorStyle()" data-toggle="dropdown"></button>
                 <ul class="dropdown-menu">
                     <div class="container col-sm-8">
                         <div data-ng-repeat="r in range(rows) track by $index" class="row">
                             <li data-ng-repeat="c in range(cols) track by $index" style="display: inline-block">
-                                <button data-ng-if="pcIndex(r, c) < pickerColors.length" class="sr-color-button" data-ng-class="{\'selected\': getColor(model[field]).toUpperCase() == getPickerColor(r, c).toUpperCase()}" data-ng-style="bgColorStyle(getPickerColor(r, c))" data-ng-click="setColor(getPickerColor(r, c))"></button>
+                                <button type="button" data-ng-if="pcIndex(r, c) < pickerColors.length" class="sr-color-button" data-ng-class="{\'selected\': getColor(model[field]).toUpperCase() == getPickerColor(r, c).toUpperCase()}" data-ng-style="bgColorStyle(getPickerColor(r, c))" data-ng-click="setColor(getPickerColor(r, c))"></button>
```

Comment:
Not for this PR, but we can ditch this limited color picker in favor of the new one